index.html

```diff
@@ -44,16 +44,142 @@ <h2 class="car">I'm a software developer</h2>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <input type="button" class="button" value="See Project">
+                    </div>
+                </div>
+                <div class="project-box">
+                    <img class="img-project2" alt="project2-image" src="img/test.png" />
+                    <div class="project-details">
+                        <h3>Multi-Post Stories Gain+Glory</h3>
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <input type="button" class="button" value="See Project">
+                    </div>
+                </div>
+                <div class="project-box">
+                    <img class="img-project3" alt="project3-image" src="img/test.png" />
+                    <div class="project-details">
+                        <h3>Multi-Post Stories Gain+Glory</h3>
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <input type="button" class="button" value="See Project">
+                    </div>
+                </div>
+                <div class="project-box">
+                    <img class="img-project4" alt="project4-image" src="img/test.png" />
+                    <div class="project-details">
+                        <h3>Multi-Post Stories Gain+Glory</h3>
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <input type="button" class="button" value="See Project">
+                    </div>   
+                </div>
+                <div class="project-box">
+                    <img class="img-project5" alt="project5-image" src="img/test.png" />
+                    <div class="project-details">
+                        <h3>Multi-Post Stories Gain+Glory</h3>
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <input type="button" class="button" value="See Project">
```

Comment:
- [ ] Kindly style your buttons to look like the Figma template, please.
*Your button*
![btn](https://user-images.githubusercontent.com/94551484/185505392-fafbbbe9-5501-4c62-a458-69de5f5be5c1.png)
*Design template*
![btn1](https://user-images.githubusercontent.com/94551484/185505400-64cf6cb2-7564-419d-aaad-fbe66d82673d.png)